### PR TITLE
[electrophysiology_uploader] insert data to raisinbread

### PR DESCRIPTION
**Pull request by GeorgeMurad**

### CODE REVIEW

File: raisinbread/RB_files/RB_electrophysiology_uploader.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO `electrophysiology_uploader` (`UploadID`,`MetaData`,`UploadDate`,`SessionID`,`UploadLocation`,`Status`,`UploadedBy`) 
+values('1','Montreal','2023-04-19 14:43:22','02','archives/MTL001_300001_V1_bids_1681247806.tar.gz','Not Started','George Murad');
```
Comment (kongtiaowang) [inline raisinbread/RB_files/RB_electrophysiology_uploader.sql:2]: set SessionID as '1' not '01'

**Comment by laemtl** [comment]
Thanks George! I think the only user that exist in RB data is admin. Can you make that changes? :)

File: raisinbread/RB_files/RB_electrophysiology_uploader.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO `electrophysiology_uploader` (`UploadID`,`MetaData`,`UploadDate`,`SessionID`,`UploadLocation`,`Status`,`UploadedBy`)
```
Comment (kongtiaowang) [inline raisinbread/RB_files/RB_electrophysiology_uploader.sql:1]: insert "SET FOREIGN_KEY_CHECKS=0;
TRUNCATE TABLE `electrophysiology_uploader `;
LOCK TABLES `electrophysiology_uploader ` WRITE; " in first line.

**Comment by kongtiaowang** [comment]
uploadby change to 'admin'

**Review by kongtiaowang — APPROVED** [review]
LGTM

**Pull request merged**